cpp/bench/matrix/argmin.cu

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <common/benchmark.hpp>
+#include <raft/matrix/math.cuh>
+
+#include <rmm/device_uvector.hpp>
+
+namespace raft::bench::linalg {
+
+struct ArgminParams {
+  int64_t rows, cols;
+};
+
+template <typename T, typename OutT>
+struct Argmin : public fixture {
+  Argmin(const ArgminParams& p)
+    : params(p), matrix(p.rows * p.cols, stream), indices(p.cols, stream)
```

Comment:
No. But actually, I've seen a few other benchmarks that leave the data uninitialized. E.g `reduce.cu`. Do you think it would be better to fill it with uniform random data?

Reply:
Yes please.